server/lib/request-handler.js

```diff
@@ -1,16 +1,33 @@
-var request = require('request');
 var path = require('path');
-
-//change db path
+var unirest = require('unirest');
 var db = require('/../../db');
-//
 
-//TODO: implement bill search handler to return full bill text
-exports.billSearch = function(req, res) {
 
+//termSearch function that utilizes a word-association API to get
+//list of word associated with user search word(s)
+exports.termSearch = function(req, res) {
+  //searchTerm format: "entry=term"
+  var searchTerm = 'entry=';
+
+  //req.body.term comes in as an array of search terms input form user
+  //convert search term array to string
+  var searchTerms = req.body['term'].slice();
+  searchTerms.toString();
+  searchTerm.concat(searchTerms);
+
+  //sends post request to Twinword API with user entered search term
+  unirest.post("https://twinword-word-associations-v1.p.mashape.com/associations/")
+    .header("X-Mashape-Key", "uSuQU8KYwRmshWTMvvi8MXcuEgn2p1IeI8BjsnPoS7M2U4ujCs")
+    .header("Content-Type", "application/x-www-form-urlencoded")
+    .header("Accept", "application/json")
+    .send(searchTerm)
+    .end(function (result) {
+      //sends word association back to client as an array of words
+      res.writeHead(200).send(result.body['associations_array']);
+  });
 }
 
 //Renders home page
 exports.renderIndex = function(req, res) {
-	res.sendFile(path.join(__dirname, '/../../client/index.html'));
-}
\ No newline at end of file
+  res.sendFile(path.join(__dirname, '/../../client/index.html'));
```

Comment:
If we're using express.static, we shouldn't need to manually handle the '/' endpoint for serving static pages. And we don't need the renderIndex method.